lib/ansible/modules/async_wrapper.py

```diff
@@ -30,11 +30,11 @@
 job_path = ''
 
 
-def notice(msg):
+def notice(msg: str):
```

Comment:
Since you're adding typing, plz include complete signature.
```suggestion
def notice(msg: str) -> None:
```